include/shark/Algorithms/QP/BoxConstrainedProblems.h

```diff
@@ -370,7 +413,8 @@ struct BoxConstrainedShrinkingProblem : public BoxConstrainedProblem<Problem>{
 	using base_type::isUpperBound;
 	using base_type::boxMin;
 	using base_type::boxMax;
-		
+	using base_type::setInitialSolution;
```

Comment:
needed iff you call the function somewhere inside BoxConstrainedShrinkingProblem